modules/mauth-common/src/main/java/com/mdsol/mauth/MAuthRequest.java

```diff
@@ -77,6 +77,9 @@ public MAuthRequest(String authenticationHeaderValue, byte[] messagePayload, Str
   public MAuthRequest(String authenticationHeaderValue, byte[] messagePayload, String httpMethod,
       String timeHeaderValue, String resourcePath, String queryParameters) {
     this(authenticationHeaderValue, messagePayload, null, httpMethod, timeHeaderValue, resourcePath, queryParameters);
+    if (messagePayload == null) {
+      this.messagePayload = new byte[] {};
+    }
```

Comment:
unnecessary if we put this logic in the primary constructor

Reply:
fixed https://github.com/mdsol/mauth-jvm-clients/pull/132/commits/16a30f1c09fed3a1590ed3b3e8cd106a367ce027